docs/book/src/development/development.md

```diff
@@ -38,9 +44,10 @@ git fetch upstream
 Build `clusterawsadm` in `cluster-api-provider-aws`:
 
 ```bash
-cd "$(go env GOPATH)"/src/sigs.k8s.io/cluster-api-provider-aws/v2
+cd "$(go env GOPATH)"/src/sigs.k8s.io/cluster-api-provider-aws/
 make clusterawsadm
-mv ./bin/clusterawsadm /usr/local/bin/clusterawsadm
+# Optional, move the binary to /usr/local/bin 
```

Comment:
I prefer this to be a move. Because whichever is first in the path will be used instead of the one on the location you copied to. That would be rather bad if you would update one but run the other.

Reply:
Ok, and I agree with that. Through the dev environment setup guide there were several steps that called the clusterawsadm binary either through ./bin/ or assuming it was moved to your $PATH. I wanted to make the documentation consistent so I changed the mv to optional but it's easily changeable where we always do a move then assume the binary is in the path. I'll make those changes shortly.